packages/commons-ui-next/src/HTMLEmbed/HTMLEmbed.js

```diff
@@ -0,0 +1,85 @@
+import { Stack } from "@mui/material";
+import React from "react";
+
+import RichTypography from "@/commons-ui/next/RichTypography";
+
+const newsletterSubscriptionStyles = (theme) => ({
```

Comment:
```suggestion
const htmlEmbedStyles = (theme) => ({
```

And may be these styles should live with the apps?